code/pizzabot.js

```diff
@@ -1,7 +1,49 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
-//Put your Javscript code here:
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice);
+
+let orderName;
+let orderQuantity;
+
+const startOrder = () => {
+  alert(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`);
+  orderName = prompt("Enter the name of the pizza you want to order today.");
+  console.log(orderName);
+  checkOrderName(orderName)
+};
+
+const checkOrderName = (orderName) => {
+  if (orderName === vegetarian || orderName === hawaiian || orderName === pepperoni) {
+    orderQuantity = prompt(`How many of ${orderName} pizza do you want?`);
+    console.log(orderQuantity);
+    showConfirmation()
+  } else {
+    alert("This pizza is not on the menu. Please try again.");
+    orderName = prompt("Enter the name of the pizza you want to order today.");
+    checkOrderName()
+  }
+};
+
+const totalCost = () => {
+  return orderQuantity * pizzaPrice
+};
+
+const getCookingTime = () => {
+  if (orderQuantity < 3) {
+    return '10';
```

Comment:
These are numbers (int) so you don't need `'` around it!
```suggestion
    return 10;
```